src/LrtSquare.sol

```diff
@@ -356,6 +426,23 @@ contract LrtSquare is
         return (totalValue * vaultTokenShares) / totalSupply;
     }
 
+    function getPercentagesInVault() public view returns (uint64[] memory) {
+        uint256 len = tokens.length;
+        uint64[] memory percentagesInVault = new uint64[](len);
+        uint256 vaultTotalValue = getVaultTokenValuesInEth(totalSupply());
+
+        for (uint256 i = 0; i < len; ) {
+            uint256 valueOfTokenInVault = getAvsTokenTotalValuesInEth(tokens[i]);
+            percentagesInVault[i] = SafeCast.toUint64((valueOfTokenInVault * HUNDRED_PERCENT_LIMIT) / vaultTotalValue);
+            unchecked {
+                ++i;
+            }
+        }
+
+        return percentagesInVault;
```

Comment:
let's return the list of addresses `tokens` together so that this API can be more useful

Reply:
Adding that